tests/functional/test_article.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+
+from pages.article import ArticlePage
+
+
+# page title
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_title_is_expected(base_url, selenium):
```

Comment:
If you'd like, not required, you can combine these test cases into grouping of logical tests, perhaps `test_article_page_header_layout()`, `test_article_page_content_layout()`, `test_article_page_footer_layout()`, etc. 

As a unit of measure, these testcases are verifying the layout of the page and logically fit together. The added benefit will be an increase in test execution, instead of a new browser instance needing to be spawned per test, only one browser session would be used.